### DISCUSSION TITLE
Does WatsonORM prevent SQL Injections by default?

### DISCUSSION THREAD

**Discussion by ghost** [jchristn/WatsonORM#7] [General]
Hallo,

i wanted to know if WatsonORM is safe from sql injection or should i verify the income values in my code before inserting it to Watson?

Thanks!

**Comment by jchristn** [comment]
Hi @flo2000ace this project uses DatabaseWrapper under the hood which performs input sanitization (see https://github.com/jchristn/databasewrapper).
